TempleApp/src/main/java/example/TempleApp/activities/InsertData.java

```diff
@@ -24,11 +25,13 @@
 
 public class InsertData extends AppCompatActivity {
 
-    String id,name,poojaTyp,overall,money,paidCheck ="NOT PAID";
+    String id,name,poojaTyp,overall,money,paidCheck ="NOT PAID", amnt;
     Spinner poojaType;
     RadioGroup radioGroup;
     LinearLayout totalLayout;
     EditText moneyDonated;
+    EditText custompuja;
```

Comment:
we should follow either "pooja" or "puja"
but "pooja" is used in the entire app it's better to use that.